examples/cr.yaml

```diff
@@ -9,7 +9,12 @@ metadata:
   # for dev:
   namespace: openshift-console
 spec:
-  paused: false
+  # openshift/api/operator/v1alpha1/types.go
+  # managementState indicates whether the operator is managing the CR (managed),
+  # ignoring the CR(paused) or if the CR is deleted (removed)
+  managementState: managed
```

Comment:
This makes more sense as a boolean. How would the `removed` value be used if the CR itself is deleted?

Reply:
Remove means clean up resources you created and pause.  Deleting the CR will cause the GC to clean up everything else.

Reply:
Why is there a distinction? That's the point of using `ownerReferences`.

Reply:
This is standardized across a bunch (all?) of the operators now.

The `ConsoleOperator` -> `Console` relationship is like a `Deployment` to a `Pod` relationship.  If you delete a pod, the deployment recreates it immediately.  the only way to actually delete is to set its `ManagementState` to `Removed`, then it will delete all the sub resources. A lot like `Replicas` `0`.